web/urls.py

```
@@ -414,6 +414,17 @@
     path("features/vote/", feature_vote, name="feature_vote"),
     path("features/vote-count/", feature_vote_count, name="feature_vote_count"),
     path("contributors/<str:username>/", views.contributor_detail_view, name="contributor_detail"),
+    # Membership URLs
+    path("membership/checkout/<int:plan_id>/", views.membership_checkout, name="membership_checkout"),
+    path(
+        "membership/create-subscription/", views.create_membership_subscription, name="create_membership_subscription"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)